plugins/connection/docker.py

```diff
@@ -22,23 +22,56 @@
           R(community.docker.docker_api,ansible_collections.community.docker.docker_api_connection)
           connection plugin.
     options:
+      remote_addr:
+        description:
+            - The name of the container you want to access.
+        default: inventory_hostname
```

Comment:
I don't think it makes sense to keep this.
```suggestion
```

Reply:
it does inform about the correct default, even if functionally it cannot ever result in the string value